hadoop-project/pom.xml

```diff
@@ -2198,7 +2227,7 @@
             <configuration>
               <rules>
                 <DependencyConvergence>
-                  <uniqueVersions>true</uniqueVersions>
+                  <uniqueVersions>false</uniqueVersions>
```

Comment:
This intentional?

Reply:
Yes, Since the thirdparty version being used is a snapshot version. If we don't disable it the build fails due to different timestamp with the snapshot version at jenkins(though it doesn't bother me at local). Once the thirdaparty version is released. Will get this back to `true` while updating the version